.babelrc

```diff
@@ -1,3 +1,6 @@
 {
+  "plugins": [
```

Comment:
because that's what we did in `site`, but I guess that was so we could also strip flow types, so i guess it doesn't matter here